express/scripts/gnav.js

```diff
@@ -146,11 +147,54 @@ function loadFEDS() {
 
   const isHomepage = window.location.pathname.endsWith('/express/');
   const isMegaNav = window.location.pathname.startsWith('/express')
-    || window.location.pathname.startsWith('/education');
+  || window.location.pathname.startsWith('/education');
   const fedsExp = isMegaNav
     ? `adobe-express/ax-gnav${isHomepage ? '-homepage' : ''}`
     : 'cc-express/cc-express-gnav';
 
+  const breadCrumbList = [];
```

Comment:
Instead of defining a variable, why not just return the array from `buildBreadCrumbArray()`?

Reply:
@toka1300 i think `buildBreadcrumbsArray()` has to be `await`ed anyway, so you can ensure the `fedsConfig` is complete by the time FEDS is loading, so i don’t see a big risk making `loadFEDS()` itself async (but not awaited when called), on the contrary, i think it will even lead to more stable behavior.

Reply:
@rofe, ok yes that makes sense. I have made the update